lib/src/spotify_base.dart

```diff
@@ -52,7 +52,15 @@ abstract class SpotifyApiBase {
   Future<Null> _refreshToken() async {
     if (_apiToken == null || _apiToken.isExpired) {
       var headers = {'Authorization': 'Basic ${_credentials.basicAuth}'};
-      var body = {'grant_type': 'client_credentials'};
+      var body;
```

Comment:
Please remove that change from the PR, since it's coming in with #28 :smile_cat: